package.json

```diff
@@ -7,25 +7,28 @@
     "bugs": "https://github.com/privacy-scaling-explorations/zk-kit/issues",
     "private": true,
     "scripts": {
-        "build": "yarn build:libraries && yarn compile:contracts",
+        "build": "turbo build",
         "build:libraries": "yarn workspaces foreach -A -t --no-private run build",
-        "compile:contracts": "yarn workspaces foreach -A run compile",
-        "test": "yarn test:libraries && yarn test:contracts && yarn test:circuits",
+        "test": "turbo test:libraries test:circuits test:coverage",
         "test:libraries": "jest --coverage",
         "test:library": "jest packages/${0}",
         "test:circuits": "yarn workspace @zk-kit/circuits test",
-        "test:contracts": "yarn workspace imt.sol test:coverage",
+        "test:contracts": "turbo test:coverage",
```

Comment:
were tests in `lazytower.sol` deliberately excluded?
One is timing out is the ci that I decided to skip instead.